sdk/appcenter-distribute/src/main/res/values/appcenter_distribute.xml

```diff
@@ -14,7 +13,12 @@
     <string name="appcenter_distribute_dialog_actioned_on_disabled_toast">Distribute was disabled</string>
     <string name="appcenter_distribute_unknown_sources_dialog_message">For security, your device is set to block installation of apps obtained from unknown sources.</string>
     <string name="appcenter_distribute_unknown_sources_dialog_settings">Settings</string>
+    <!-- TODO: Add translations for the alias. -->
```

Comment:
We don't need to since it's aliased, I just tested alias follows translations (tested I was seeing french for android 4.4 progress dialog title).
What we need to do later is remove the old key and rename the translated keys back to the main one in 3.0 so the TODO here is inaccurate.